tests/unit/test_source_detection_step2.py

```
@@ -0,0 +1,124 @@
+    print(f"\n🔍 Verificando dependencias:")
+    
+    try:
+        import subprocess
+        result = subprocess.run(['ffmpeg', '-version'], capture_output=True, text=True, timeout=5)
+        if result.returncode == 0:
+            print(f"   ✅ FFmpeg: Disponible")
+            ffmpeg_available = True
+        else:
+            print(f"   ❌ FFmpeg: No disponible")
+            ffmpeg_available = False
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)